packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -871,9 +872,9 @@ function MapAnalysisImpl(props: ImplProps) {
                   <MapSidePanel
                     isExpanded={appState.isSidePanelExpanded}
                     isUserLoggedIn={userLoggedIn}
-                    onToggleIsExpanded={() =>
-                      setIsSidePanelExpanded(!appState.isSidePanelExpanded)
-                    }
+                    onToggleIsExpanded={() => {
+                      setIsSidePanelExpanded(!appState.isSidePanelExpanded);
+                    }}
```

Comment:
```suggestion
                    onToggleIsExpanded={() =>
                      setIsSidePanelExpanded(!appState.isSidePanelExpanded)
                    }
```

I'd go back to the original. Maybe you added logging here but best to not make unnecessary changes.

Reply:
complied!